docs/migrations/web/MIGRATION-v3.md

```diff
@@ -6,9 +6,163 @@ Introducing version 3 of the _spirit-web_ package.
 
 ## Overview
 
+- [General Changes](#general-changes)
+  - [Themes](#themes)
+  - [Spacing values](#spacing-values)
+  - [Helpers: Typography `text` infix and Heading emphasis](#helpers-typography-text-infix-and-heading-emphasis)
+  - [Utilities: Removed and new classes](#utilities-removed-and-new-classes)
+  - [Migrating product components](#migrating-product-components)
 - [Component Changes](#component-changes)
   - [Button: renamed modifier `square`](#button-square-modifier-renamed-to-symmetrical)
 
+## General Changes
+
+## Themes
+
+Starting from version 3, the design system now supports multiple themes based on the
+theme tokens. As in Figma, the first theme is the default one.
+
+The default theme is set to the `:root` element and all the themes are generated
+to classes that can be applied to the any element. Their names are based on the
+theme names in tokens.
+
+This change affects all components that apply any color.
+
+We did all the work and our components are now using CSS variables to
+apply colors, however, if you have any custom styles you need to update them.
+
+Also, all your components that previously used color tokens should be updated to
+use CSS variables. See [Migrating product components](#migrating-product-components)
+for more information.
+
+To switch the theme, simply add the theme class to the element you want to apply the theme to.
+All the colors in the nested component will be switched to the theme colors.
+
+Using themes we were able to remove `inverted` variants and some `brand` tokens.
+
+See our [Themes decision][themes-decision] for more information and why we decided to implement this feature.
+
+### Spacing values
+
+New spacing values were added to the design system this caused a change in the
+spacing tokens and some values were updated.
+
+All spacing props and utility classes are influenced by this change.
```

Comment:
I think this can be said in one sentence:

```suggestion
New spacing values were added to the design system, updating spacing design tokens and affecting all spacing props and utility classes.
```

Or something like that.